mlrun/projects/project.py

```diff
@@ -3761,6 +3767,79 @@ def _validate_file_path(self, file_path: str, param_name: str):
                 f"<project.spec.get_code_path()>/<{param_name}>)."
             )
 
+    def _resolve_artifact_producer(
+        self,
+        artifact: typing.Union[str, Artifact, DatasetArtifact, ModelArtifact],
+        project_producer_tag=None,
```

Comment:
I do because of https://github.com/mlrun/mlrun/pull/5283/files#diff-c972cfb48585c0ee85184b83808a9bf592a0ddd60a07cba789023649354f6cb8R1378
I want all artifacts in the list to have the same project tag